docs/how-to/configure-jenkins-memory-usage.md

```diff
@@ -0,0 +1,12 @@
+# How to control heap memory of the jenkins-k8s-operator charm
+The [jenkins-k8s-operator](https://github.com/canonical/jenkins-k8s-operator) charm uses [juju constraints](https://juju.is/docs/juju/constraint) to limit the amount of memory a charm can use. To deploy the charm with constraints, use the `--constraints "<key>=<value>"` option when running `juju deploy`:
+```bash
+juju deploy jenkins-k8s --channel=latest/edge --constraints "mem=2048M"
+```
+To change this value after deployment, use the `set-constraints` command.
+```bash
+juju set-constraints jenkins-k8s "mem=4096M"
+```
+Other types of constraints (like cores, disk, etc.) can also be applied. Note that this value affects the shared maximum memory between the `charm` container and `jenkins` container.
+# Considerations when applying memory constraints
```

Comment:
Perhaps leave a blank line before a heading? Generally markdown should be nice to read in the text based format as well as the rendered format 🙂

Reply:
got it, I added a new line before the heading